tests/ossfuzz/fuzz_common.h

```diff
@@ -0,0 +1,43 @@
+#include <inttypes.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include "lzma.h"
+
+
+// Output buffer for decompressed data. This is write only; nothing cares
+// about the actual data written here.
+static uint8_t OUTBUF[4096];
+
+// Some header values can make liblzma allocate a lot of RAM
+// (up to about 4 GiB with liblzma 5.2.x). We set a limit here to
+// prevent extreme allocations when fuzzing.
+const uint64_t MEM_LIMIT = 300 << 20; // 300 MiB
+
+
+void fuzz_code(lzma_stream *stream, const uint8_t *inbuf, size_t inbuf_size) {  
```

Comment:
The function name should start on a new line. And this should be static. So the first line:
static void

Second line:
fuzz_code(...

This is to match our style in the project. Having the function name starting at column index 0 helps certain tools and text editors search for function names.